junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/extension/DisabledCondition.java

```diff
@@ -39,15 +39,16 @@
 	 */
 	@Override
 	public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext context) {
-		Optional<AnnotatedElement> element = context.getElement();
-		Optional<Disabled> disabled = findAnnotation(element, Disabled.class);
-		if (disabled.isPresent()) {
-			String reason = disabled.map(Disabled::value).filter(StringUtils::isNotBlank).orElseGet(
-				() -> element.get() + " is @Disabled");
-			return ConditionEvaluationResult.disabled(reason);
-		}
-
-		return ENABLED;
+		AnnotatedElement element = context.getElement().orElse(null);
```

Comment:
Why the `orElse`?

Reply:
Because 
```java
.orElseGet(() -> element.get() + " is disabled via @org.junit.Ignore"))
```
look ugly imho and IDE highlights `element.get()` as possible nullability problem